hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/utils/HddsVersionInfo.java

```diff
@@ -50,7 +50,9 @@ public static void main(String[] args) {
         "Compiled with protoc " + HDDS_VERSION_INFO.getProtocVersion());
     System.out.println(
         "From source with checksum " + HDDS_VERSION_INFO.getSrcChecksum());
-    LOG.debug("This command was run using " +
-        ClassUtil.findContainingJar(HddsVersionInfo.class));
+    if (LOG.isDebugEnabled()) {
+      LOG.debug("This command was run using " +
```

Comment:
Use {}

Reply:
Hi @bharatviswa504, because of the **if** condition check, the expression evaluation is guaranteed so eager or late didn't matter, hence did not make these changes.